src/character.cpp

```diff
@@ -1014,6 +1014,31 @@ std::vector<item_location> Character::find_ammo( const item &obj, bool empty, in
     return res;
 }
 
+std::vector<item_location> Character::find_reloadables()
+{
+    std::vector<item_location> reloadables;
+
+    visit_items( [this, &reloadables]( item *node ) {
```

Comment:
```suggestion
    visit_items( [this, &reloadables]( item * node ) {
```

Reply:
Gah, these files got astyled while the PR was pending, hopefully this will be the last time I have to deal with that...